README.md

```diff
@@ -150,6 +156,26 @@ GRAFANA_TEST_DB=mysql go test ./pkg/...
 GRAFANA_TEST_DB=postgres go test ./pkg/...
 ```
 
+#### End to end
+
+Execute all end to end tests
+
+```bash
+yarn e2e-tests
+```
+
+Execute all end to end tests using using a specific url
+
+```bash
+ENV BASE_URL=http://localhost:3333 yarn e2e-tests
+```
+
+Debugging all end to end tests
+
+```bash
+ENV BROWSER=1 SLOWMO=1 yarn e2e-tests
```

Comment:
Maybe we could explain in detail what those envs do?

Reply:
Good catch, I'll do that.